app/_locales/en/messages.json

```diff
@@ -2329,6 +2329,9 @@
     "message": "verify the network details",
     "description": "Serves as link text for the 'unrecognizedChain' key. This text will be embedded inside the translation for that key."
   },
+  "unsendableAsset": {
+    "message": "Sending collectible (erc-721) tokens is not currently supported"
```

Comment:
Nit: "ERC" is usually capitalized, and this message could use a description to help translators understand what it's for.